common/src/main/java/org/apache/falcon/util/DateUtil.java

```diff
@@ -39,8 +47,37 @@
 
     public static final long HOUR_IN_MILLIS = 60 * 60 * 1000;
 
+    private static final Pattern GMT_OFFSET_COLON_PATTERN = Pattern.compile("^GMT(\\-|\\+)(\\d{2})(\\d{2})$");
+
+    public static final TimeZone UTC = getTimeZone("UTC");
+
+    public static final String ISO8601_UTC_MASK = "yyyy-MM-dd'T'HH:mm'Z'";
+    private static String activeTimeMask = ISO8601_UTC_MASK;
+    private static TimeZone activeTimeZone = UTC;
+
+    private static final Pattern VALID_TIMEZONE_PATTERN = Pattern.compile("^UTC$|^GMT(\\+|\\-)\\d{4}$");
+
+    private static final String ISO8601_TZ_MASK_WITHOUT_OFFSET = "yyyy-MM-dd'T'HH:mm";
+    private static boolean entityInUTC = true;
+
     private DateUtil() {}
 
+    /**
+     * Configures the Datetime parsing with Oozie processing timezone.
+     *
+     */
+    public static void setTimeZone(String tz) throws FalconException {
```

Comment:
As we have to support EL expression supported by oozie, i have copied from oozie code base and added here